cpp/buffs.cpp

```diff
@@ -12,7 +12,7 @@ Buff buff_list[] =
   {  13452, {   0,  25,   0, 0, 28, 0,  0,  0,  0,  0,  0, 0 } }, // Elixir of the Mongoose
   {  22831, {   0,  35,   0, 0, 20, 0,  0,  0,  0,  0,  0, 0 } }, // Elixir of Major Agility
   {  27664, {   0,  20,   0, 0, 0, 0,  0,  0,  0,  0,  0, 0 } },  // Grilled Mudfish
-  {  27658, {   0,   0,   0, 0, 0, 0,  0,  0,  0,  0,  0, 0 } },  // Roasted Clefthoof
+  {  27658, {   20,   0,   0, 0, 0, 0,  0,  0,  0,  0,  0, 0 } },  // Roasted Clefthoof
```

Comment:
All of the .cpp files are actually deprecated -- it took me a while to understand it, so you can see I have some new stuff added that doesn't actually matter. All of the buffs are done in buffs.js. 

I need to remove the .cpp entirely except for main, and probably remove the C++ -> JS conversion package as well. I've been putting off dealing with the build infrastructure.